src/i18n/strings/en_EN.json

```diff
@@ -2477,6 +2477,14 @@
     "Got an account? <a>Sign in</a>": "Got an account? <a>Sign in</a>",
     "New here? <a>Create an account</a>": "New here? <a>Create an account</a>",
     "Notification settings": "Notification settings",
+    "Get your own Element!": "Get your own Element!",
+    "Confirm Abort Of Host Creation": "Confirm Abort Of Host Creation",
+    "Are you sure you wish to abort creation of the host? The process cannot be continued.": "Are you sure you wish to abort creation of the host? The process cannot be continued.",
+    "Abort": "Abort",
+    "Failed to connect to your homeserver. Please close this dialog and try again.": "Failed to connect to your homeserver. Please close this dialog and try again.",
+    "Set up your own personal Element host": "Set up your own personal Element host",
+    "Please accept transfer of data to the Element Pro setup wizard to continue. The setup wizard will be able to access your account for the duration of the setup process.": "Please accept transfer of data to the Element Pro setup wizard to continue. The setup wizard will be able to access your account for the duration of the setup process.",
```

Comment:
Do we need all these strings even marked as translatable as they don't relate to any actual Element features?

Reply:
so long as it is a string in the project - yes.